modules/rostests/winetests/iphlpapi/iphlpapi.c

```diff
@@ -2538,9 +2538,9 @@ static void convert_luid_to_name( NET_LUID *luid, WCHAR *expect_nameW, int len )
     }
 #ifdef __REACTOS__
     if (prefix)
-        _snwprintf( expect_nameW, len, L"%s_%d", prefix, luid->Info.NetLuidIndex );
+        _snwprintf( expect_nameW, len, L"%s_%d", prefix, (int)luid->Info.NetLuidIndex );
```

Comment:
Are these changes related to `ROS_LONG64` (see https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/winetests/iphlpapi/CMakeLists.txt;hb=51d7aeb3ea40f8c291041c09e4a72d901a07e8cc#l2 ) ?
I would invite you to remove that define from the cmakelists and see what happens if you don't add these casts.